scripts/globals/status.lua

```diff
@@ -941,6 +941,8 @@ MOD_UDMGRANGE         = 390
 MOD_CRITHITRATE       = 165
 MOD_CRIT_DMG_INCREASE = 421
 MOD_ENEMYCRITRATE     = 166
+MOD_MAGIC_CRITHITRATE = 541
+MOD_MAGIC_CRIT_DMG_INCREASE = 541
```

Comment:
These shouldn't be the same number. Also, since I forgot to increment the spare mods at the bottom, the next unused modifier value is actually 562 (I redid elemental staves so they took a LOT of mods). So use 562/563 please.